modules/jjwg_Maps/views/view.geocoded_counts.php

```diff
@@ -39,15 +39,20 @@ public function display()
             echo '<br /><br />';
         }
 
-        echo '<table cellspacing="0" cellpadding="0" border="0" class="list view" style="width: 50% !important;"><tbody>';
+        echo '<div class="list-view-rounded-corners">';
+
+        echo '<table cellspacing="0" cellpadding="0" border="0" class="list view" style="width: 50% !important;">';
+        echo '<thead>';
         echo '<tr><th>'.$GLOBALS['mod_strings']['LBL_MODULE_HEADING'].'</th>';
         foreach ($this->bean->geocoded_headings as $heading) {
-            echo '<th>'.$heading.'</th>';
+            echo '<th>'.$GLOBALS['mod_strings']['LBL_'.$heading].'</th>';
```

Comment:
We should consider adding in some way to identify what labels we are building dynamically (Potentially just leaving a comment in the language file?). Ideally want to prevent them from appearing to be unused.